### ISSUE TITLE
macOS build is broken

### ISSUE THREAD

**Issue by hluk** [hluk/CopyQ#2450]
The bundling fails with:

```
+ cpack
CPack: Create package using DragNDrop
CPack: Install projects
CPack: - Run preinstall target for: copyq
CPack: - Install project: copyq []
CMake Error at /usr/local/Cellar/cmake/3.27.4/share/cmake/Modules/BundleUtilities.cmake:458 (message):
  otool -l failed: 1

  error:
  /Applications/Xcode_13.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool-classic:
  can't open file:
  @loader_path/../../../../../opt/freetype/lib/libfreetype.6.dylib (No such
  file or directory)

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.27.4/share/cmake/Modules/BundleUtilities.cmake:527 (get_item_rpaths)
  /usr/local/Cellar/cmake/3.27.4/share/cmake/Modules/BundleUtilities.cmake:614 (set_bundle_key_values)
  /usr/local/Cellar/cmake/3.27.4/share/cmake/Modules/BundleUtilities.cmake:933 (get_bundle_keys)
  /Users/runner/work/CopyQ/build/src/cmake_install.cmake:190 (fixup_bundle)
  /Users/runner/work/CopyQ/build/cmake_install.cmake:62 (include)
```

I tried few workarounds but none worked:
- `brew unlink freetype && brew link freetype`
- Use older version of qt@5 - this would need much more work then just using the older `qt@5.rb`
- Move to Qt 6 - KDE Framework still does not support it

**Comment by hluk** [comment]
I moved back to using `macdeployqt` instead of `cpack` and it seems to work well (at least in the build environment): https://github.com/hluk/CopyQ/actions/runs/6065215399

**Comment by chenrui333** [comment]
we did rebottle for freetype per this PR, https://github.com/Homebrew/homebrew-core/pull/141250, can you give freetype a new try?

**Comment by hluk** [comment]
> we did rebottle for freetype per this PR, [Homebrew/homebrew-core#141250](https://github.com/Homebrew/homebrew-core/pull/141250), can you give freetype a new try?

I tested it yesterday. The problem with freetype was fixed. Unfortunately, the error popped up for another library (libsharpyuv) - see https://github.com/Homebrew/homebrew-core/issues/140930#issuecomment-1704039107.

**Comment by Paulchen-Panther** [comment]
The libsharpyuv dependency comes from the QT imageplugin libqwebp. If you don't need this, the problem can be solved relatively quickly.
Comment: https://github.com/Lord-Grey/hyperion.ng/commit/83b434a75fb3106191d71a6d061f9f85daa7d772

If you need the imageplugin, you can fix the dependency by passing the path (FIXUP_BUNDLE_DEP_DIRS) (/opt/homebrew/Cellar/webp/1.3.x/lib) to fixup_bundle.

If all this is not a solution for you, I hope to be able to help with a little informat.

Greetings Paulchen

**Comment by hluk** [comment]
> Nice to see this closed. Does this also fix cover #1884 ?

No, unfortunately, there is no M1 support.
